docroot/themes/humsci/humsci_basic/src/scss/components/_pattern.testimonial.scss

```diff
@@ -78,27 +108,63 @@
     }
   }
 
-  &__author-credits {
-    margin-bottom: hb-calculate-rems(20px);
-  }
-
   &__name {
     font-weight: hb-theme-font-weight(semibold);
+
+    @include hb-themes(('colorful', 'airy')) {
+      &,
+      .hb-dark-pattern .hs-paragraph-style--well &,
+      .hb-dark-pattern .hs-paragraph-style--well-full-width & {
+        color: var(--palette--primary);
+      }
+    }
+
+    @include hb-traditional {
+      &,
+      .hb-dark-pattern .hs-paragraph-style--well &,
+      .hb-dark-pattern .hs-paragraph-style--well-full-width & {
+        color: var(--palette--secondary);
+      }
+    }
+
+    .hb-dark-pattern & {
+      color: var(--palette--white);
+    }
   }
 
   &__position {
     font-size: hb-calculate-rems(16px);
-
+    
```

Comment:
Please avoid adding lines with just spaces. This can be prevented setting auto formatting with prettier in your editor.